.travis.yml

```diff
@@ -4,3 +4,15 @@ go: "1.11"
 install: true
 script:
 - GINKGO_EXTRA_ARGS="--compilers=2" make all test/unit
+
+deploy:
+- provider: script
+  script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make handler push-handler
```

Comment:
Humm do we need it for stdci so we don't build there ?

Reply:
you want to do it in the stdci instead of travis?

Reply:
No no, I was meaning that now that we push "master" containers we may want to consume then at stdci so we don't need to build there, but if we consume there at stdci.

But thinking about using or not travis to release, we may want to stay away from github for this and do it with stdci or even prow in case we move to that.